app/controllers/vendor_api/decisions_controller.rb

```diff
@@ -67,7 +67,7 @@ def application_choice
     end
 
     def render_application
-      render json: { data: SingleApplicationPresenter.new(application_choice).as_json }
+      render json: { data: SingleApplicationPresenter.new(application_choice.reload).as_json }
```

Comment:
Why is this needed?

Reply:
Because the application_choice object is not updated as part of Make or ChangeOffer service to include `offer` (as the association is not setup and the offer object is created separately), which results in the API returning out of date information. We can remove when the association is implemented.

Reply:
Maybe we could add some TODOs that mention the Trello card in which we'll add the association?

Reply:
I would reference from trello rather than from the code, as I don't think we do go through TODOs, so it's just going to end up being more noise that we ignore.

Reply:
https://trello.com/c/Cn4e6z2P/1588-add-offer-association-to-applicationchoice-model